Tests/StoreKitUnitTests/StoreKit2/StoreKit2TransactionFetcherTests.swift

```diff
@@ -148,6 +148,78 @@ class StoreKit2TransactionFetcherTests: StoreKitConfigTestCase {
         expect(result) == transaction
     }
 
+    // MARK: - receipt
+
+    func testGeneratesReceipt() async throws {
+        _ = try await self.createTransaction(productID: Self.product1,
+                                             finished: false)
+        _ = try await self.createTransaction(productID: Self.product2,
+                                             finished: false)
+        let receipt = await self.fetcher.receipt
+        expect(receipt.transactions.count) ==  2
+        expect(receipt.subscriptionStatus.count) == 2
```

Comment:
There's a better way to write this that provides more informative error messages:
```suggestion
        expect(receipt.transactions).to(haveCount(2))
        expect(receipt.subscriptionStatus).to(haveCount(2))
```